src/core/GLBufferAttribute.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * @author raub / https://github.com/raub
+ */
+
+
+function GLBufferAttribute( gl, buffer, type, itemSize, count, normalized ) {
+
+	this.sizes = [
```

Comment:
Is this property supposed to be private? Is so I would remove it from the public API.

Reply:
The object stored in `this.sizes` is used by methods below. I also need `gl` to build this object. This is why I build it at the construction time, when `gl` is by definition available. Being a 'public' member is just a side-effect of the above. What would you suggest?

Reply:
It seems you need `sizes ` only to determine `elementSize`, right? Check my comment below. I think it's better if the user defines `elementSize`.